packages/web-client/app/utils/web3-strategies/kovan.ts

```diff
@@ -79,6 +81,26 @@ export default class KovanWeb3Strategy implements Layer1Web3Strategy {
     }
   }
 
+  async disconnect(): Promise<void> {
+    if (this.currentProviderId === 'metamask') {
```

Comment:
@lukemelia Metamask doesn't seem to have a good way to disconnect from the Dapp side. Wondering if we should show a specific message to tell the user to disconnect from the Dapp?

Reply:
Ideally, yes, but as long as we remove the connection from the dapp side, it's OK for now I think.

Reply:
Will leave it like this for now, and bring this up with the design team in the next meeting, just in case there are specific requirements/user expectations.